sunpy/timeseries/metadata.py

```diff
@@ -450,25 +463,29 @@ def values(self):
 
     @property
     def time_range(self):
-        """Returns the TimeRange of the entire time series meta data."""
+        """
+        Returns the `~sunpy.time.TimeRange` of the entire timeseries metadata.
+        """
         start = self.metadata[0][0].start
         end = self.metadata[0][0].end
         for metatuple in self.metadata:
             if end < metatuple[0].end:
-               end = metatuple[0].end
+                end = metatuple[0].end
         return TimeRange(start, end)
 
     def _remove_columns(self, colnames):
-        """Removes the given column/s from the TimeSeriesMetaData object.
+        """
+        Removes the given column(s) from the
```

Comment:
no line break here?

Reply:
Doc Formater got there. I can fix it. Noticed another mistake.